ideascube/tests/helpers.py

```diff
@@ -0,0 +1,39 @@
+from django.apps import apps
+from django.test import TransactionTestCase
+from django.db.migrations.executor import MigrationExecutor
+from django.db import connection
+import pytest
+
+needMigration = pytest.mark.skipif(
+    pytest.config.getoption("--nomigrations"),
+    reason="need migrations to run tests on migrations"
+)
+
+@needMigration
```

Comment:
Since you're using the decorator only once, why introduce this intermediary variable instead of directly using the `pytest.mark.skipif`?

Also... camelCase? o_O